src/pages/Models/Details/ModelDetails.js

```diff
@@ -95,6 +101,20 @@ const filterModelStatusData = (modelStatusData, appName) => {
   return modelStatusData;
 };
 
+const generateTerminalComponent = (modelUUID, controllerWSHost) => {
+  return null; // XXX Remove me to see the Terminal
```

Comment:
The Terminal UI will be disabled until the 2.9 release of Juju and the dashboard.